folly/detail/PolyDetail.h

```diff
@@ -32,6 +32,12 @@
 
 #include <folly/PolyException.h>
 
+#if defined(__cpp_template_auto) || defined(__cpp_nontype_template_parameter_auto)
+#define FOLLY_NTTP_AUTO 1
```

Comment:
Namespacing: let's use the name `FOLLY_POLY_NTTP_AUTO`.

Reply:
Good call, done!

Reply:
Oops misspelled poly in my first commit 😅